tests/unit/backend/common/utils/dl_sources/test_url.py

```diff
@@ -1,107 +0,0 @@
-import os
-import unittest
-from unittest import TestCase
-from urllib.parse import urlparse
-
-import boto3
-import requests
-from moto import mock_s3
-
-from backend.common.utils.dl_sources.url import from_url, RegisteredSources, S3URL, DropBoxURL, S3URI
```

Comment:
This code is still used in url_provider.py, the test should be updated rather than deleted